### PR TITLE
Collected additional necessary information for each of the event reco…

### PR DESCRIPTION
**Pull request by ShaneKent**
…rds. Also changed the timeline coloring. Delete the config.json file in your working directory to have the new one take effect.

### CODE REVIEW

File: winlogtimeline/collector/parser.py [hunk 1]
```diff
@@ -51,115 +51,259 @@ def parser(raw, record):
...
-    record["description"] = "Logoff (Network Connection)"
+    data = raw["Event"]["EventData"]["Data"]
+
+    record["account"] = data[4]["#text"]
+    record["description"] = "Logoff"
+
+    if data[4]["#text"] == "3":
+        record["details"] = "End of Network Connection session"
+    else:
+        record["details"] = "EXCLUDED"
```
Comment (carterv) [inline winlogtimeline/collector/parser.py]: Checkout out this branch locally and ran it. All log off events show up as 'EXCLUDED'. Is this expected?

File: winlogtimeline/ui/ui.py [hunk 2]
```diff
@@ -135,13 +135,17 @@ def __init__(self, parent, headers, data, **kwargs):
...
         # Determine the column widths
         for row in data:
             for i, v in enumerate(row):
+                print(i, self.headers[i], v)
+                if self.headers[i] == "Details":
+                    continue
                 width = font.Font().measure(v)
                 if width > col_width[self.headers[i]]:
                     col_width[self.headers[i]] = width
+        """
```
Comment (carterv) [inline winlogtimeline/ui/ui.py]: I would still like to revisit this and try and speed it up at a later date.

Comment (ShaneKent) [reply winlogtimeline/ui/ui.py]: I agree.

File: winlogtimeline/collector/parser.py (hunk 1 above)
Comment (ShaneKent) [reply winlogtimeline/collector/parser.py]: screen shot 2017-12-01 at 11 27 05 am
I'm only matching what the Excel Macro has.

**Pull request merged**